charts/opensearch-dashboards/values.yaml

```
@@ -155,3 +155,16 @@ nodeSelector: {}
 tolerations: []
 
 affinity: {}
+
+lifecycle: {}
+  # preStop:
+    # exec:
+  #     command: ["/bin/sh", "-c", "echo Hello from the postStart handler > /usr/share/message"]
+  # postStart:
+  #   exec:
```

```suggestion
  postStart: {}
  #   exec:
```

Uncomment this and maybe add a comment above linking to the lifecycle page: https://kubernetes.io/docs/concepts/containers/container-lifecycle-hooks/#container-hooks

---

@DandyDeveloper I think it's good to document the usage as you have suggested with the link however I'd still leave it up to the user to make sure of the yaml object. The commented section is only there as an example